sql/atxs/atxs.go

```diff
@@ -442,13 +447,18 @@ func Add(db sql.Executor, atx *types.ActivationTx, blob types.AtxBlob) error {
 			stmt.BindNull(13)
 		}
 		stmt.BindInt64(14, int64(atx.Weight))
+		if atx.MarriageATX != nil {
+			stmt.BindBytes(15, atx.MarriageATX.Bytes())
+		} else {
+			stmt.BindNull(15)
```

Comment:
Is `BindNull` needed to set a column to `NULL`? I think it would also work without.

Reply:
Apparently, it's not needed, at least tests pass, although I couldn't find it documented anywhere.

Reply:
The sqlite docs say:

>Unbound parameters are interpreted as NULL.

So I guess it's not needed to explicitly bind NULL.